react-app/dist/index.html

```diff
@@ -0,0 +1,11 @@
+<html>
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
```

Comment:
Starting from React v15, IE8 is [no longer supported](https://reactjs.org/blog/2016/01/12/discontinuing-ie8-support.html), so it doesn't make much sense to keep this tag.